apps/sports-helsinki/src/pages/api/readiness.ts

```diff
@@ -1,5 +1,16 @@
 import type { NextApiRequest, NextApiResponse } from 'next';
 
 export default function handler(_: NextApiRequest, res: NextApiResponse) {
-  res.status(200).json({ status: 'ok' });
+  const packageVersion = process.env.APP_VERSION;
+  const release = process.env.NEXT_PUBLIC_RELEASE;
+  const commitHash = process.env.NEXT_PUBLIC_COMMITHASH;
+  const buildTime = process.env.BUILD_TIME;
+
+  res.status(200).json({
+    status: 'ok',
+    release: `${release}`,
+    packageVersion: `${packageVersion}`,
+    commitHash: `${commitHash}`,
+    buildTime: `${buildTime}`,
+  });
```

Comment:
The apostrophes are not needed -- not even the value is needed.

```suggestion
const packageVersion = process.env.APP_VERSION ?? '';
const release = process.env.NEXT_PUBLIC_RELEASE ?? '';
const commitHash = process.env.NEXT_PUBLIC_COMMITHASH ?? '';
const buildTime = process.env.BUILD_TIME ?? '';

res.status(200).json({
  status: "ok",
  release,
  packageVersion,
  commitHash,
  buildTime,
});
```

Reply:
refactored